### ISSUE TITLE
Accessibility of search button text

### ISSUE THREAD

**Issue by nevanscott** [ironhack-edu/orange-search-engine-project#7]
Either the value should be set to Search, or consider using a button with a type of submit, and Text which reads Search. Use CSS to visually hide the text so it is still accessible to screen readers.

https://github.com/ironhack-edu/orange-search-engine-project/blob/5c017c7b0ebf9cf9796586827f03035a1520012c/iteration-4-final/index.html#L58
